csi.proto

```diff
@@ -496,6 +503,51 @@ message ControllerServiceCapability {
 }
 ////////
 ////////
+message NodePublishDeviceRequest {
```

Comment:
1. should this message include the volume "attributes", like the other publish calls?
2. should this message include staging credentials? (plugin can authn/z the RPC call)

Reply:
1. Yes, I will add
2. Yes, I will add, also for Unstage